docker-compose.dev.yml

```diff
@@ -7,7 +7,7 @@ services:
     image: mealie-frontend:dev
     build:
       context: ./frontend
-      dockerfile: Dockerfile.frontend
```

Comment:
> Small fix regarding the renamed Dockerfile.frontend used by docker-compose.dev.yml.

The dockerfile isn't setup for development so it won't hot reload. Currently the docker-compose.dev.yml file isn't functional. 

See These two issues for reference

- https://github.com/hay-kot/mealie/pull/1072
- https://github.com/hay-kot/mealie/pull/756

Ideally this would get fixed, but I ran into some issues and didn't have time to track them all down.